skbio/stats/distance/tests/test_base.py

```diff
@@ -57,6 +64,110 @@ def setUp(self):
                                    np.array(self.dm_2x2_asym_data),
                                    np.array(self.dm_3x3_data)]
 
+    def test_within(self):
+        exp = pd.DataFrame([['a', 'a', 0.0],
+                            ['a', 'c', 4.2],
+                            ['c', 'a', 4.2],
+                            ['c', 'c', 0.0]],
+                           columns=['i', 'j', 'value'])
+        obs = self.dm_3x3.within(['a', 'c'])
+        pdt.assert_frame_equal(obs, exp)
+
+    def test_within_order_stability(self):
+        exp = pd.DataFrame([['a', 'a', 0.0],
+                            ['a', 'c', 4.2],
+                            ['c', 'a', 4.2],
+                            ['c', 'c', 0.0]],
+                           columns=['i', 'j', 'value'])
+
+        # NOTE: order was changed from ['a', 'c'] to ['c', 'a']
+        # but the output order in exp is consistent with
+        # test_within
+        obs = self.dm_3x3.within(['c', 'a'])
+        pdt.assert_frame_equal(obs, exp)
```

Comment:
Can you also include:
```
        obs = self.dm_3x3.within(['a', 'c'])
        pdt.assert_frame_equal(obs, exp)
```
In this test, similar to `test_between_order_stability`, just to assure permutation stability within the test.